### PR TITLE
feat(Dgraph): Add ability to change size of caches through the admin interface.

### PR DESCRIPTION
**Pull request by martinmr**
The endpoints have been changed from lruMb to cacheMb to reflect that this
is changing the value of the cacheMb flag.

Fixes 2515

---
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/6644)

 
 
 

Docs Preview: [Dgraph Preview](https://dgraph-ba131887fb-100128.surge.sh)

### CODE REVIEW

**Comment by martinmr** [comment]
CI is failing because the cache sizes need to be properly initialized. Once branch `martinmr/wal` is merged into master, CI should start working again.

**Review by manishrjain — APPROVED** [review]
:lgtm: Ensure build is still green after this.

> Reviewed 16 of 16 files at r1.
> *[Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/6644#-:-MJ2SoiQE6WLiTNoi5_U:b-fu04wq)* status: all files reviewed, 1 unresolved discussion (waiting on @danielmai, @martinmr, @MichaelJCompton, @pawanrawal, and @vvbalaji-dgraph)

___
*[worker/worker.go, line 131 at r1](https://reviewable.io/reviews/dgraph-io/dgraph/6644#-MJ2RsrtD6HHlxKb7I1X:-MJ2RsrtD6HHlxKb7I1Y:b2wkxtz) ([raw file](https://github.com/dgraph-io/dgraph/blob/1bfce228edfe1f58956de10eae70f9e79a4be24a/worker/worker.go#L131)):*
> ```Go
> // UpdateCacheMb updates the value of cache_mb and updates the corresponding cache sizes.
> func UpdateCacheMb(memoryMB int64) error {
> 	glog.Infof("memoryMB %d", memoryMB)
> ```

Updating cacheMB to ...

**Pull request merged**
